src/projetos/HomeProjetosListPage.tsx

```
@@ -325,21 +206,13 @@ export function HomeProjetosListPage() {
                       <LoadingCard />
                     </div>
                   ))
-                ) : projetosAtuais.length > 0 ? (
-                  projetosAtuais.map((projeto) => (
-                    <div
-                      key={projeto.id}
-                      className="flex-shrink-0"
-                      style={{ width: '500px' }}
-                    >
+                ) : paginatedItems.length > 0 ? (
+                  paginatedItems.map((projeto: any) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)